### ISSUE TITLE
CHALLENGE:  Design a survey that has multiple geo questions (geopoint, geoshape, geotrace) in the same survey and set different default basemaps at the geo question level. After publishing, verify that it is persisted in field app and web app

### ISSUE THREAD

**Issue by BrandonArmstrong** [tedrick/Survey123-Testing#248]
## Question level basemap defaults

To set a different default basemap for each **geopoint, geotrace** and **geoshape** question in a survey, use a "map name" parameter in the body::esri:style column in the XLSform. You may need to manually add the body::esri:style column to your existing XLSform if it is not present. Check out the Advanced template in Connect for all the latest updates.

The parameter syntax simply uses the map name, and the "map name" must be wrapped in quotations to ensure it is handled correctly when the name contains spaces:

`map="World Imagery"`

The above parameter will set the default to the World Imagery basemap for that specific geopoint question only. The other geopoint, geotrace or geoshape questions in the same survey will still use the default World Street Map basemap if they don't have a parameter set.

You can also set offline map packages as the default basemap using the body::esri:style parameter, using the exact title of the package as you see it in the Linked Content settings:

`map="My Offline MMPK"`

Currently, only the **"map name"** syntax is supported in the parameter to specify the default, but we have a future enhancement to use the **"itemID"** of a basemap or web map instead.

This will make it easier when you have multiple maps named the same in the different folders in your organization.

By using a combination of the survey level default basemap settings and the question level basemap settings you can fully customize which questions use which basemaps, and ensure the basemap being used is fit for purpose for the workflow and activity being undertaken in the field app.

**Comment by billwie** [comment]
I"m unable to publish a map with multiple Geo questions. if this turns out to be a bug then i'll log it as a separate issue.

**Comment by billwie** [comment]
> I"m unable to publish a map with multiple Geo questions. if this turns out to be a bug then i'll log it as a separate issue.

After James T straightened me out I was able to publish with multiple geo questions and get different basemaps on each. the only problem i encountered was that for some reason "World Topographic Map" shows as streetmap instead. works in Windows, iOS and Android.

**Comment by NeilMYoung** [comment]
I was able to successfully publish, a survey with a geopoint in the main body of the form, with two repeats. One was a line and one was a poly. worked fine outside when testign both on and offline. I have been attempting to send. I hit send and it appears to not have an error, but the items stay in the outbox. One of the times I got the attached message, however most of the time nothing happens - they just stay in the outbox without an error message.
